core/src/test/java/io/aiven/klaw/service/TopicSyncControllerServiceTest.java

```diff
@@ -1,38 +1,32 @@
 package io.aiven.klaw.service;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyInt;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.when;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
 
 import io.aiven.klaw.UtilMethods;
 import io.aiven.klaw.config.ManageDatabase;
-import io.aiven.klaw.dao.Env;
-import io.aiven.klaw.dao.KwClusters;
-import io.aiven.klaw.dao.Team;
-import io.aiven.klaw.dao.UserInfo;
+import io.aiven.klaw.dao.*;
```

Comment:
We are removing these wild card imports, and rather have individual imports. Its a setting in Intellij. I have defined that as 20 in my settings.

Reply:
I went through and removed all .* imports in the class.